File: codegen/codegen-erlang/src/test/resources/ir/runtime_helpers_resolve_base_url.expected.erl
-spec resolve_base_url(map()) -> binary().
resolve_base_url(Config) ->
    Prefix = maps:get(endpoint_prefix, Config),
    Region = maps:get(region, Config, <<"us-east-1">>),
    <<"https://", Prefix/binary, ".", Region/binary, ".amazonaws.com">>.

File: codegen/codegen-elixir/src/test/resources/ir/runtime_helpers_resolve_base_url.expected.ex
@spec resolve_base_url(map()) :: String.t()
def resolve_base_url(config) do
  prefix = Map.fetch!(config, :endpoint_prefix)
  region = Map.get(config, :region, "us-east-1")
  "https://#{prefix}.#{region}.amazonaws.com"
end
